src/config/playpenconfig.rs

```diff
@@ -0,0 +1,68 @@
+use std::path::{PathBuf, Path};
+
+use super::tomlconfig::TomlPlaypenConfig;
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+pub struct PlaypenConfig {
+    editor: PathBuf,
```

Comment:
> All those points are great but not related to this PR.

Exactly. That is what I meant when talking about refactor in another PR's. Imho this PR should not be blocked on more refactoring. 
Further rewrites should be given some thought.